azkaban-exec-server/src/main/java/azkaban/execapp/FlowPreparer.java

```diff
@@ -41,19 +41,17 @@
 public class FlowPreparer {
```

Comment:
I agree but by that logic, we may have to change the visibility of a lot of classes. I left it defaulted to public for now since that is the usual convention for classes.